doc/develop/getting_started/index.rst

```diff
@@ -621,6 +621,13 @@ that are used to emulate, flash and debug Zephyr applications.
             You must rerun the setup script if you relocate the Zephyr SDK bundle directory after
             the initial setup.
 
+#. :ref:`Set these environment variables <env_vars>` as described `here
+   <toolchain_zephyr_sdk>`::
+
+   - Set :envvar:`ZEPHYR_TOOLCHAIN_VARIANT` to ``zephyr``
+   - Set :envvar:`ZEPHYR_SDK_INSTALL_DIR` to the toolchain installation
+     directory
```

Comment:
Adding this as `.. note::` would make more sense IMHO since this is not a hard requirement.

Reply:
I think I disagree. There have been a lot of fights in the past about too many `.. note::` directives being added to the GSG, with the result being that it is not readable.

Reply:
I should have started the comment above with "if any."

Use of the "note directive" would be totally justified here since this is only supplementary information. As for whether this belongs in the GSG is questionable as discussed above.

Reply:
I think a note is appropriate in this case, since it doesn't apply to all users equally. See my note proposal @mbolivar-nordic and @stephanosio.